docs/els-for-runtimes-and-libraries/jsonpath-plus/README.md

```diff
@@ -0,0 +1,135 @@
+
+* JSONPath Plus 7.2.0
+
+## Connection to ELS for JSONPath Plus Library
+
+This guide outlines the steps needed to integrate the TuxCare ELS for the JSONPath Plus library.
+
+## Step 1: Get Token
+
+You need a token in order to use TuxCare ELS JSONPath Plus library. Anonymous access is disabled. To receive the token, please contact [sales@tuxcare.com](mailto:sales@tuxcare.com).
+
+## Step 2: Set Up ELS for JSONPath Plus
+
+TuxCare provides ELS for JSONPath Plus as an NPM package, hosted on a secure internal registry. Follow the steps below to add it to your project and get started.
+
+1. Navigate to the root directory of your JSONPath Plus project.
+2. Create a `.npmrc` file or update it if it already exists.
+
+   **Example:**
+
+   ```text
+   my-jsonpath-plus-project/
+   ├── node_modules/
+   ├── package.json
+   ├── .npmrc         ⚠️ ← Create it here
+   └── package-lock.json
+   ```
+
+3. Use an editor of your choice (e.g., VS Code) to add the following registry address line:
+
+   <CodeWithCopy>
+
+   ```text
+   registry=https://registry.npmjs.org/
+   @els-js:registry=https://nexus.repo.tuxcare.com/repository/els_js/
+   //nexus.repo.tuxcare.com/repository/els_js/:_auth=${TOKEN}
+   ```
+
+   </CodeWithCopy>
+
+   :::warning
+   Replace ${TOKEN} with the token you received from [sales@tuxcare.com](mailto:sales@tuxcare.com).
+   :::
+
+4. Update your `package.json` file to replace your JSONPath Plus dependencies with the TuxCare packages:
+
+   <CodeWithCopy>
+
+   ```text
+   "dependencies": {
+     "jsonpath-plus": "npm:@els-js/jsonpath-plus@7.2.0-tuxcare.1"
+   }
+   ```
+
+   </CodeWithCopy>
+
+5. You need to remove the `node_modules` directory and the `package-lock.json` file, and also clear the `npm cache` before installing the patched packages. Use the following commands:
+   
+   <CodeWithCopy>
+
+   ```text
+   rm -rf node_modules package-lock.json && npm cache clean --force
+   ```
+
+   </CodeWithCopy>
+
+6. Run the following command to install the ELS version of the JSONPath Plus library (token for the TuxCare repository will be automatically picked up from your `.npmrc` file):
+
+   <CodeWithCopy>
+
+   ```text
+   npm install
+   ```
+
+   </CodeWithCopy>
+
+   You will see an output like:
+
+   ```text
+   added 15 packages, and audited 16 packages in 2s
+
+   1 package is looking for funding
+   run `npm fund` for details
+
+   found 0 vulnerabilities
+   ```
+
+7. You've successfully installed the Tuxcare ELS version of the JSONPath Plus library into your project.
+
+## Vulnerability Exploitability eXchange (VEX) 
+
+VEX is a machine-readable format that tells you if a known vulnerability and is actually exploitable in your product. It reduces false positives, helps prioritize real risks.
+
+TuxCare provides VEX for JSONPath Plus ELS versions: [security.tuxcare.com/vex/cyclonedx/els_lang_javascript/jsonpath-plus/](https://security.tuxcare.com/vex/cyclonedx/els_lang_javascript/jsonpath-plus/).
```

Comment:
404
should be 
https://security.tuxcare.com/vex/cyclonedx/els_lang_javascript/jsonpath/